tests/e2e/iothub_e2e/client_fixtures.py

```diff
@@ -52,9 +42,8 @@ def sastoken_ttl(request):
 
 
 @pytest.fixture(scope="function")
-def client_kwargs(connection_retry, websockets, keep_alive, sastoken_ttl):
+def client_kwargs(websockets, keep_alive, sastoken_ttl):
```

Comment:
Technically, `sastoken_ttl` is probably not going to be a keyword argument, but rather an optional positional argument. Haven't fully decided yet, but that's currently how it is.

That said, don't change this yet, this is purely an informative comment. I've still got to iron out a couple SAS-related API things after bugbash, but before GA.